mat-process-api.Tests/V1/Validators/GetProcessRequestValidatorTest.cs

```diff
@@ -0,0 +1,53 @@
+using Bogus;
+using FluentValidation.TestHelper;
+using mat_process_api.V1.Validators;
+using NUnit.Framework;
+
+namespace mat_process_api.Tests.V1.Validators
+{
+
+    [TestFixture]
+    public class GetProcessRequestValidatorTest
+    {
+        GetProcessDocumentRequestValidator validator;
+        private readonly Faker faker = new Faker();
+
+        [SetUp]
+
+        public void SetUp()
+        {
+            validator = new GetProcessDocumentRequestValidator();
+        }
+
+        [Test]
+        [TestCase(null)]
+        [TestCase("")]
+        [TestCase(" ")]
+        public void given_null_or_empty_or_whitespace_processRef_the_validator_should_return_an_error(string processRef)
```

Comment:
If you will want to test error messages you'll have to split 'null' test into separate [Test].